crates/meroctl/src/cli/context/mod.rs

```diff
@@ -0,0 +1,57 @@
+use clap::{Parser, Subcommand};
+use reqwest::Url;
+
+use crate::cli::RootArgs;
+
+mod create;
+mod ls;
+mod members;
+mod mutate;
+mod query;
+
+#[derive(Debug, Parser)]
+pub struct ContextCommand {
+    #[command(subcommand)]
+    pub subcommand: ContextSubCommands,
+}
+
+#[derive(Debug, Subcommand)]
+pub enum ContextSubCommands {
+    Ls(ls::LsCommand),
```

Comment:
for the sanity of the type system, let's use `List` here, with an alias for `ls`